models/users_model.js

```diff
@@ -18,6 +18,18 @@ const selectAllUsers = function(next) {
     });
 };
 
+const deleteUser = function({email}, next) {
+    const query = "DELETE FROM development.users WHERE email IN (?)";
+    const params = [email];
+
+    client.execute(query, params, {prepare: true}, (err) => {
+        if (err) {
+            return next(err);
+        }
+    });
+    next(null);
```

Comment:
This should be inside the callback function, otherwise it will call next before the query is finished.

Reply:
Good catch